containers/ddev-router/Dockerfile

```diff
@@ -6,22 +6,50 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV DOCKER_GEN_VERSION 0.7.4
 ENV DOCKER_HOST unix:///tmp/docker.sock
 
+# TARGETPLATFORM is Docker buildx's target platform (e.g. linux/arm64), while 
+# BUILDPLATFORM is the platform of the build host (e.g. linux/amd64)
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+
+SHELL ["/bin/bash", "-c"]
+
 RUN apt-get -qq update && \
     apt-get -qq install --no-install-recommends --no-install-suggests -y \
         ca-certificates certbot curl less python-certbot-nginx procps telnet vim wget && \
     apt-get autoremove -y && \
     apt-get clean -y && \
 	rm -rf /var/lib/apt/lists/*
 
-ADD https://github.com/jwilder/forego/releases/download/v0.16.1/forego /usr/local/bin/forego
+RUN if [[ $TARGETPLATFORM == "linux/arm64" ]]; then \
+        # jwilder/forego doesn't have an arm64 download so we resort to the original forego repo
+        wget -q https://bin.equinox.io/c/ekMN3bCZFUn/forego-stable-linux-arm64.tgz && \
+        tar -C /usr/local/bin -xvzf forego-stable-linux-arm64.tgz && \
+        rm /forego-stable-linux-arm64.tgz && \
+        chmod ugo+x /usr/local/bin/forego; \
+    else \
+        curl -sSL https://github.com/jwilder/forego/releases/download/v0.16.1/forego -o /usr/local/bin/forego && \
+        chmod ugo+x /usr/local/bin/forego; \
+    fi
 
-RUN wget -q https://github.com/jwilder/docker-gen/releases/download/$DOCKER_GEN_VERSION/docker-gen-linux-amd64-$DOCKER_GEN_VERSION.tar.gz && \
-    tar -C /usr/local/bin -xvzf docker-gen-linux-amd64-$DOCKER_GEN_VERSION.tar.gz && \
-    rm /docker-gen-linux-amd64-$DOCKER_GEN_VERSION.tar.gz && \
-    chmod ugo+x /usr/local/bin/forego && \
-    mkdir -p /etc/nginx/certs /mnt/ddev-global-cache/mkcert
+RUN if [[ $TARGETPLATFORM == "linux/arm64" ]]; then \
```

Comment:
I think it would probably be best to group all the arm64  in a single if/else instead of doing it multiple times.